app/src/main/java/ch/hikemate/app/ui/components/CustomTextField.kt

```diff
@@ -0,0 +1,75 @@
+package ch.hikemate.app.ui.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.selection.TextSelectionColors
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.OutlinedTextFieldDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.text.style.TextAlign
+import ch.hikemate.app.ui.theme.primaryColor
+
+object CustomTextField {
+  const val MAX_NAME_LENGTH = 40
+  const val MAX_EMAIL_LENGTH = 50
+  const val MAX_PASSWORD_LENGTH = 50
+}
```

Comment:
Question: How have you defined those constants?

Reply:
Completly arbitrary. Do you have a better value in mind ?

Reply:
Nop, it was just to know!